modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -1247,12 +1345,18 @@ struct InferList2: public cv::detail::KernelTag {
 
         const auto &input_name_0 = uu.params.input_names.front();
         const auto &mm_0 = in_metas[0u];
-        const auto &matdesc = cv::util::get<cv::GMatDesc>(mm_0);
+
+        if (!(cv::util::holds_alternative<cv::GMatDesc>(mm_0) ||
+              cv::util::holds_alternative<cv::GFrameDesc>(mm_0))) {
+            util::throw_error(std::runtime_error(
+                        "IE Backend: Unsupported input meta"
```

Comment:
It's `OV` backend, not `IE`

Reply:
Done, thanks! https://github.com/opencv/opencv/pull/24938/commits/2f249a8acab382376637a71aaba2940544eebd45